### PR TITLE
feat(luxand plugin): add luxand Face SDK plugin

### PR DESCRIPTION
**Pull request by molobala**
This plugin add support for Luxand Face SDK

### CODE REVIEW

File: src/@ionic-native/plugins/luxand/index.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+ * - Remove this note
+ *
+ */
+import { Injectable } from '@angular/core';
+import { Cordova, IonicNativePlugin, Plugin } from '@ionic-native/core';
+
+/**
+ * @name Luxand
+ * @description
+ * This plugin does something
```
Comment (danielsogl) [inline src/@ionic-native/plugins/luxand/index.ts]: Add a description

File: src/@ionic-native/plugins/luxand/index.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+ * ```typescript
+ * import { Luxand } from '@ionic-native/luxand';
+ *
+ *
+ * constructor(private luxand: Luxand) { }
+ *
+ * ...
+ *
+ *
+ * this.luxand.functionName('Hello', 123)
```
Comment (danielsogl) [inline src/@ionic-native/plugins/luxand/index.ts]: Add an example

File: src/@ionic-native/plugins/luxand/index.ts [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+ *   .then((res: any) => console.log(res))
+ *   .catch((error: any) => console.error(error));
+ *
+ * ```
+ */
+@Plugin({
+  pluginName: 'Luxand',
+  plugin: 'codova-plugin-luxand', // npm package name, example: cordova-plugin-camera
```
Comment (danielsogl) [inline src/@ionic-native/plugins/luxand/index.ts]: Remove boilerplate comments

File: src/@ionic-native/plugins/luxand/index.ts [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  installVariables: [], // OPTIONAL the plugin requires variables
+  platforms: ['Android', 'iOS'] // Array of platforms supported, example: ['Android', 'iOS']
+})
+@Injectable()
+export class Luxand extends IonicNativePlugin {
+
+  /**
+   * Initialize Luxand SDK
+   * @param config {LuxandConfig} Some param to configure something
+   * @return {Promise<any>} Returns a promise that resolves when something happens
```
Comment (danielsogl) [inline src/@ionic-native/plugins/luxand/index.ts]: Remove boilerplate comments here and the other functions

File: src/@ionic-native/plugins/luxand/index.ts [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+   * clearMemory method, try to clear internal database
+   * @return {Promise<OMLFacialData>} Returns a promise that resolves when something happens
+   */
+  @Cordova()
+  clearMemory(): Promise<OMLFacialData> {
+    return; // We add return; here to avoid any IDE / Compiler errors
+  }
+
+}
+export interface LuxandConfig {
```
Comment (danielsogl) [inline src/@ionic-native/plugins/luxand/index.ts]: Move interfaces to the top

**Comment by molobala** [comment]
Hello,

fixed now.

**Pull request merged**